addons/mail/static/src/messaging/entity/chat_window/chat_window.js

```diff
@@ -30,7 +16,14 @@ function ChatWindowFactory({ Entity }) {
          * Close this chat window.
          */
         close() {
+            // don't update foldState for "new message" chat window (no thread)
+            const thread = this.thread;
             this.delete();
+            if (thread) {
+                // called after delete because thread _updateAfter code depends
+                // on chatWindow existence
```

Comment:
Would just remove comments, it's too specific with current state of code, so very likely to be misleading in the future. And without comment, it's easy to understand.

Reply:
ok